pages/_app.tsx

```diff
@@ -1,12 +1,38 @@
 import type { AppProps } from "next/app";
-import GlobalStyle from "../styles";
+import { ThemeProvider } from "@mui/material/styles";
+import CssBaseline from "@mui/material/CssBaseline";
+import { createTheme } from "@mui/material/styles";
+
+const theme = createTheme({
```

Comment:
I am new to MUI, I assume here i can override the standard MUI settings

Reply:
Yep exactly